### PR TITLE
No eval

### LINKED ISSUE #311: CSP violation: Unsafe-eval
### Expected behavior and actual behavior.
I'd like to use this library without having to weaken the Content Security Policy of our application.

It seems like `eval` is used in the source code of this library (`component.js` line 150, `main.js` line 25). This isn't a good practice because of the potential XSS-vulnerabilities that come along with it.

On this note I would like to see information about the security against XSS-vulnerabilities of this package in the `README.md`.

### Steps to reproduce the problem.

Our CSP: 
```
default-src 'self';script-src 'self';frame-src 'self' blob:;frame-ancestors 'self';worker-src 'self';object-src 'none';font-src 'self';style-src 'self' 'unsafe-inline';img-src 'self' data: blob:;media-src 'self' blob:;upgrade-insecure-requests
```

The Error:
```
Refused to evaluate a string as JavaScript because 'unsafe-eval' is not an allowed source of script in the following Content Security Policy directive: "script-src 'self'".
...
    at window.canvasDatagrid.window.require.window.canvasDatagrid (canvas-datagrid.js:1)

```

### PR DESCRIPTION
**Pull request by josh-hemphill**
Fixes #311 

Changes proposed in this pull request:

- [x] Replace function created from `eval` with `Function()` constructor
- [ ] Replace class obfuscation created with `eval`?

I haven't been able to decipher what the other eval is actually doing to be able to look at replacing it.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,109 +1,109 @@
 {
-  "name": "canvas-datagrid",
```
Comment (ndrsn) [inline package.json:2]: Indentation here is off, again — can you restore the original indentation, so the diff shows properly?

File: webpack.config.js [hunk 2]
```diff
@@ -1,4 +1,5 @@
 const path = require('path');
+const webpack = require('webpack');
```
Comment (ndrsn) [inline webpack.config.js:2]: Why do we need to import webpack? I'm not sure it's needed for the config file — it's not being used anywhere?

File: esFill.js [hunk 3]
```diff
@@ -0,0 +1,8 @@
+const exports = {
```
Comment (ndrsn) [inline esFill.js:1]: What does this file do?

**Review by ndrsn — CHANGES_REQUESTED** [review]

File: webpack.config.js (hunk 2 above)
Comment (josh-hemphill) [reply webpack.config.js:2]: Sorry it's a fragment, I had to re-delete the webpack ESM export that I had experimented with.

File: esFill.js (hunk 3 above)
Comment (josh-hemphill) [reply esFill.js:1]: Yep, sorry, another fragment, I made a branch off of an older commit.

**Comment by ndrsn** [comment]
Thanks for this, @josh-hemphill !

**Pull request merged**